website/docs/guides/docs/docs-create-doc.mdx

```diff
@@ -83,3 +83,19 @@ With <code>{#custom-id}</code> syntax you can set your own header id.
 
 </BrowserWindow>
 ```
+
+## Doc tags {#doc-tags}
+
+Optionally, you can add tags to your doc pages, which introduces another dimension of categorization in addition to the [docs sidebar](./sidebar.md). Tags are passed in the front matter as a list of labels:
+
+<!-- prettier-ignore-start -->
+```yml "your-doc-page.md"
+---
+id: doc-with-tags
+title: A doc with tags
+tags:
```

Comment:
I've just tested Docusaurus `beta.5` and this also works fine:

```
tags: [Demo, Getting Started]
```

Do you only want to show one way, or do you want to show both the list-format _and_ the array-format?

Reply:
Um, they are both valid YAML formats anyways, not sure if the purpose here is to educate writing valid YAML objects

Reply:
Oh I never knew that the Markdown frontmatter is supposed to be a valid YAML object. I learned something new today... 😄 

I'll let you decide what you like to do here. 😉